src/helix-ui/elements/HXTooltipElement.js

```diff
@@ -132,8 +129,9 @@ export class HXTooltipElement extends _ProtoClass {
             return false;
         }
 
-        // FIXME: getRootNode() will not return document context before connect
-        return (this.getRootNode().activeElement === ctrl);
+        if (this.isConnected) {
```

Comment:
I don't think we need the `isConnected` guard here. Please revert, but remove the comment.